src/counter_v2.cairo

```diff
@@ -60,7 +60,7 @@ pub mod CounterV2 {
 
     #[derive(Copy, Drop, Debug, PartialEq, starknet::Event)]
     pub struct counter_increase_count_by_one {
```

Comment:
You should always use the [PascalCasing naming convention for structs and enums](https://book.cairo-lang.org/ch06-01-enums.html?highlight=convention#enum-variants-and-values). Rename this struct to `CountIncreased`. And also apply this change to all your `struct` and `enum` instances

Reply:
Noted 
Will attend to these now